tests/parametric/test_otel_env_vars.py

```diff
@@ -0,0 +1,190 @@
+        assert resp["dd_trace_enabled"] == "true"
+        assert resp["dd_runtime_metrics_enabled"]
+        tags = resp["dd_tags"]
+        assert "foo:bar" in tags
+        assert "baz:qux" in tags
+        assert resp["dd_trace_propagation_style"] == "b3,tracecontext"
+        assert resp["dd_trace_debug"] == "false"
+
+    @missing_feature(context.library < "nodejs@5.11.0", reason="Implemented in v5.11.0, v4.35.0, &v3.56.0")
+    @pytest.mark.parametrize(
+        "library_env",
+        [
+            {
+                "OTEL_SERVICE_NAME": "otel_service",
+                "OTEL_LOG_LEVEL": "error",
+                "OTEL_TRACES_SAMPLER": "traceidratio",
+                "OTEL_TRACES_SAMPLER_ARG": "0.1",
+                "OTEL_METRICS_EXPORTER": "none",
+                "OTEL_RESOURCE_ATTRIBUTES": "foo=bar1,baz=qux1",
+                "OTEL_PROPAGATORS": "b3,datadog",
+            }
+        ],
+    )
+    def test_otel_env_vars_set(self, test_agent, test_library):
+        resp = test_library.get_tracer_config()
+
+        assert resp["dd_service"] == "otel_service"
+        assert resp["dd_log_level"] == "error"
+        assert resp["dd_trace_sample_rate"] == "0.1"
+        assert resp["dd_trace_enabled"] == "true"
+        assert resp["dd_runtime_metrics_enabled"] == "false"
+        tags = resp["dd_tags"]
+        assert "foo:bar1" in tags
+        assert "baz:qux1" in tags
+        assert resp["dd_trace_propagation_style"] == "b3,datadog"
+
+    @missing_feature(context.library < "nodejs@5.11.0", reason="Implemented in v5.11.0, v4.35.0, &v3.56.0")
+    @pytest.mark.parametrize(
+        "library_env",
+        [
+            {
+                "OTEL_RESOURCE_ATTRIBUTES": "deployment.environment=test1,service.name=test2,service.version=5,foo=bar1,baz=qux1"
+            }
+        ],
+    )
+    def test_otel_attribute_mapping(self, test_agent, test_library):
+        resp = test_library.get_tracer_config()
+
+        assert resp["dd_service"] == "test2"
+        assert resp["dd_env"] == "test1"
+        assert resp["dd_version"] == "5"
+        tags = resp["dd_tags"]
+        assert "foo:bar1" in tags
+        assert "baz:qux1" in tags
+
+    @missing_feature(context.library < "nodejs@5.11.0", reason="Implemented in v5.11.0, v4.35.0, &v3.56.0")
+    @pytest.mark.parametrize(
+        "library_env", [{"OTEL_TRACES_SAMPLER": "always_on",}],
+    )
+    def test_otel_traces_always_on(self, test_agent, test_library):
+        resp = test_library.get_tracer_config()
+        assert resp["dd_trace_sample_rate"] == "1.0" or "1"
+
+    @missing_feature(context.library < "nodejs@5.11.0", reason="Implemented in v5.11.0, v4.35.0, &v3.56.0")
+    @pytest.mark.parametrize(
+        "library_env", [{"OTEL_TRACES_SAMPLER": "always_off",}],
+    )
+    def test_otel_traces_always_off(self, test_agent, test_library):
+        resp = test_library.get_tracer_config()
+        assert resp["dd_trace_sample_rate"] == "0.0" or "0"
```

Comment:
This will always evaluate to True 
```suggestion
        assert float(resp["dd_trace_sample_rate"]) == 0.0
```